beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -1631,6 +1631,31 @@ impl<T: BeaconChainTypes> Worker<T> {
                 // Do not propagate these messages.
                 self.propagate_validation_result(message_id, peer_id, MessageAcceptance::Ignore);
             }
+            SyncCommitteeError::PastSlot {
+                message_slot,
+                earliest_permissible_slot,
+            } => {
+                /*
+                 * This error can be triggered by a mismatch between our slot and the peer.
+                 *
+                 *
+                 * The peer has published an invalid consensus message, _only_ if we trust our own clock.
+                 */
+                trace!(
+                    self.log,
+                    "Sync committee message is not within the last MAXIMUM_GOSSIP_CLOCK_DISPARITY slots";
+                    "peer_id" => %peer_id,
+                    "type" => ?message_type,
+                );
+
+                // We tolerate messages that were just one slot late.
+                if *message_slot + 1 > *earliest_permissible_slot {
```

Comment:
I think this should be `message_slot + 1 < earliest_permissible_slot` shouldn't it?

If the message is late the earliest permissible slot will be the current slot, and we want to check that the message slot is less than `current slot - 1`

Reply:
```suggestion
                if *message_slot + 1 < *earliest_permissible_slot {
```
You are right